test/e2e/testSuite/src/basic.test.ts

```diff
@@ -269,3 +271,90 @@ test("pimlico_sendCompressedUserOperation can bundle multiple compressed userOps
         await publicClient.getBytecode({ address: relayer.account.address })
     ).toEqual(simpleAccountDeployedBytecode)
 })
+
+test("eth_sendUserOperation supports bundling with the same sender and different nonce keys", async () => {
```

Comment:
worth doing another test for being able to queue up nonce values?

Reply:
Definitely! Waiting for William's PR for updating v7 support in tests
https://github.com/pimlicolabs/alto/pull/212